central-service/services/problem.go

```diff
@@ -1,14 +1,22 @@
 package services
 
 import (
+	"github.com/acmutd/bsg/central-service/constants"
 	"github.com/acmutd/bsg/central-service/models"
 	"gorm.io/gorm"
+	"gorm.io/gorm/clause"
 )
 
 type ProblemService struct {
 	db *gorm.DB
 }
 
+type DifficultyParameter struct {
+	NumEasyProblems int
+	NumMediumProblems int
+	NumHardProblems int
+}
```

Comment:
Is this something that we intend to leave up to the users to configure, or is this something that is going to be unmodifiable? Just curious what our intent is regarding the ability to allow users to choose the number of problems they wish to have.

Reply:
This is something that we would leave up to the users to configure, but adding a default value could be a great idea too.